system/metis/values.yaml

```diff
@@ -7,7 +7,7 @@ global:
 
 alerts:
   prometheus: "infra-collector"
-  supportGroup: "observability"
+  supportGroup: "containers"
 
 owner-info:
   support-group: observability
```

Comment:
```suggestion
  support-group: containers
```

Reply:
This is another occurrence in the `alerts` directive.